### PR TITLE
Update documentation and wording on CLI

### PR DESCRIPTION
**Pull request by h4ckm03d**

### CODE REVIEW

File: envsync.go [hunk 1]
```diff
@@ -45,9 +45,11 @@ type Syncer struct {
 // e.g: FOO=bar.
 // FOO is the key and bar is the value.
 //
+// Before the synchronization process will create .bak from target file as backup
 // During the synchronization process, there may be an error.
-// Any key-values that have been synchronized before the error occurred is kept in target.
-// Any key-values that haven't been synchronized because of an error occurred is ignored.
+// If error occured, file target will restored from .bak file that created before.
```
Comment (sharonrh) [inline envsync.go]: If error occurred, target file will be restored from previously created .bak file.

File: envsync.go [hunk 2]
```diff
@@ -45,9 +45,11 @@ type Syncer struct {
...
+// If synchronization success the key-values will be sorted and added header based on prefix
+// e.g: FOO_BAR will added '#FOO' header before first key of FOO_
```
Comment (sharonrh) [inline envsync.go]: If the synchronization succeed, the key-values will be sorted and header will be added based on key's prefix
e.g: for FOO_BAR, '#FOO' header will be added before the first key of FOO_

File: README.md [hunk 3]
```diff
@@ -33,9 +33,12 @@ Envsync is a tool to synchronize sample env and actual env file.
...
 
+- With flag `-s` and `-t`
 ```
 envsync -s <source file> -t <target file>
 ```
 
-Source file is the sample env. If the -s flag isn't provided, envsync will use the default value which is **env.sample**.
-Target file is the actual env. If the -t flag isn't provided, envsync will use the default value which is **.env**.
\ No newline at end of file
+- Without flag, by default will use file `env.sample` as source and `.env` as a target on current directory.
```
Comment (sharonrh) [inline README.md]: kalimatnya lebih simpel, jadi lebih jelas. cuma jadi terkesan harus pake harus pakai kedua flag sekaligus atau tidak sama sekali ngga ya?

Comment (h4ckm03d) [reply README.md]: mungkin diberi keterangan optional flag

**Pull request merged**